karafka.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.2'
 
   spec.add_dependency 'ruby-kafka', '= 0.3.17'
-  spec.add_dependency 'sidekiq', '~> 4.2'
+  spec.add_dependency 'sidekiq', '~> 5.0'
```

Comment:
Is this bump completely necessary to add this feature? This forces everyone to upgrade to 5.0 without using any of the features required by sidekiq.